docs/topics/gradle.md

```diff
@@ -256,13 +256,12 @@ kotlin.stdlib.default.dependency=false
 
 ### Set dependencies on test libraries
 
-The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for testing different Kotlin projects. 
-
-Add the corresponding dependencies on test libraries:
-
-* For `commonTest`, add the `kotlin-test-common` and `kotlin-test-annotations-common` dependencies.
-* For JVM targets, use `kotlin-test-junit` or `kotlin-test-testng` for the corresponding asserter implementation and annotations mapping.
-* For Kotlin/JS targets, add `kotlin-test-js` as a test dependency. 
+The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for testing Kotlin projects on different Kotlin platforms.
```

Comment:
"different Kotlin platforms" doesn't tell exactly what platforms you mean. Please write precisely. "All supported platforms" I suppose.

Reply:
Supported by whom? 
"all platforms supported by Kotlin" I would say.

Reply:
"**Kotlin** projects on all supported platforms" implies that platforms are supported by Kotlin. No need for more words here.

Reply:
Indeed) Thank you!